src/components/navbar/NavBar.tsx

```diff
@@ -1,76 +0,0 @@
-import React, { useState } from 'react';
```

Comment:
Is this deletion intentional?

Reply:
No, is this bad? Should I get it back? It still works...

Reply:
Hmm, not sure. There's probably two navbars.

Reply:
Was deleted because useState is not used on the page